Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,48 @@
+class ClickhouseOdbc < Formula
+  desc "ClickHouse ODBC driver"
+  homepage "https://clickhouse.yandex"
+  url "https://github.com/yandex/clickhouse-odbc.git",
+    :tag => "v1.0.0.20180903",
+    :revision => "17ecca3993e164d2d49360ed9be68becc11b7b05"
```

Comment:
Please build from source tarball `
https://github.com/yandex/clickhouse-odbc/archive/2018-09-03.tar.gz` rather that checkout

Reply:
We use submodules but source tarball have no submodules.